nixos/release.nix

```diff
@@ -108,7 +108,7 @@ let
     let
       configEvaled = import lib/eval-config.nix {
         inherit system;
-        modules = [ module versionModule ];
+        modules = [ configuration module versionModule ];
```

Comment:
Looks like `[ configuration module versionModule ]` should be factored out across the entire file.